base/base/StringRef.h

```diff
@@ -185,7 +185,8 @@ inline bool memequalWide(const char * p1, const char * p2, size_t size)
     {
         case 3: if (!compare8(p1 + 32, p2 + 32)) return false; [[fallthrough]];
         case 2: if (!compare8(p1 + 16, p2 + 16)) return false; [[fallthrough]];
-        case 1: if (!compare8(p1, p2)) return false;
+        case 1: if (!compare8(p1, p2)) return false; [[fallthrough]];
+        default: ;
```

Comment:
With added `no_warning(switch-default)` this change doesn't seem to be relevant?

Reply:
Yes, but no problem - let's have this anyway.